astropy/table/tests/test_item_access.py

```diff
@@ -251,6 +251,22 @@ def test_np_integers(self, table_data):
         t = table_data.Table(table_data.COLS)
         idxs = np.random.randint(len(t), size=2)
         item = t[idxs[1]]
+        
+    def test_np_scalar(self, table_data):
+        """
+        Select rows using numpy scalars.
+        """
+        t = table_data.Table(table_data.COLS)
+        idx = np.int64(np.random.randint(len(t)))
+        item = t[idx]
```

Comment:
they are not the same.
In `test_np_integers`, the type of idx is `int`. Actually I don't understand why that test exists.
In this test, the type of idx is `np.int64`.